docs/extending/extending_draftail.md

```diff
@@ -422,6 +436,7 @@ Here is an example with highlighting of problematic punctuation – first, regis
 
 ```python
 from wagtail.admin.rich_text.editors.draftail.features import DecoratorFeature
+from wagtail import hooks
```

Comment:
I recommend you just add these docs improvements as a separate PR. Will be easy to merge and avoids confusion on this PR.

Reply:
Yeah, I am making the PR from a different branch. I don't know why it is appearing to be on the same branch. Here is the link: https://github.com/wagtail/wagtail/pull/11057